botorch/sampling/qmc.py

```diff
@@ -136,7 +136,7 @@ def __init__(
         )
         # compute Cholesky decomp; if it fails, do the eigendecomposition
         try:
-            self._corr_matrix = torch.cholesky(cov).transpose(-1, -2)
+            self._corr_matrix = torch.linalg.cholesky(cov).transpose(-1, -2)
         except RuntimeError:
             eigval, eigvec = torch.symeig(cov, eigenvectors=True)
```

Comment:
symeig can be replaced with linalg.eigh
```suggestion
            eigval, eigvec = torch.linalg.eigh(cov)
```